paddle/fluid/operators/detail/grpc_client.h

```diff
@@ -165,6 +165,20 @@ class FetchBarrierProcessor : public BaseProcessor {
   std::unique_ptr<sendrecv::SendRecvService::Stub> stub_;
 };
 
+class CheckpointNotifyProcessor : public BaseProcessor {
```

Comment:
Can we reuse SendProcessor?

Reply:
We can reuse ```SendProcessor``` at currently, but ```checkpoint``` may have some new features in the future, I am not sure we can use ```SendProcessor ``` all the time, so, I think we can use a new one.